src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
Please do not mark this as obsolete because *it isn't obsolete*. The methods works 100% perfectly fine as-is. If someone doesn't want to pass in a logger, that's fine. This is the pattern we've used elsewhere.